website/docs/d/logic_app_workflow.html.markdown

```diff
@@ -59,6 +59,18 @@ The following attributes are exported:
 
 * `workflow_outbound_ip_addresses` - The list of outgoing ip addresses of workflow.
 
+* `identitiy` - An `identitiy` block as defined below.
+
+---
+
+The `identity` block exports the following:
+
+* `type` - The Type of Managed Identity assigned to this Logic App Workflow. Possible values are `SystemAssigned` and `UserAssigned`.
```

Comment:
since this is a Data Source this is a read-only value, so the possible values aren't required here fwiw:

```suggestion
* `type` - The Type of Managed Identity assigned to this Logic App Workflow.
```